app/views/pages/get-started/working-with-the-code-and-files.liquid

```diff
@@ -60,7 +61,7 @@ pos-cli env add dev --email myemail@example.com —-url https://my-instance.stag
 After running this command, you’ll get asked for your Partner Portal password to authenticate. Providing the correct password will be confirmed by pos-cli:
 
 ```bash
-Environment <your instance url> as <environment> has been added successfully.
```

Comment:
Not universal.

Reply:
same as above

Reply:
This is more tricky, as the user needs to actually provide his actual email address and his URL, using our URL will cause Unauthorized error :) Probably not worth it, but one potential solution that coms to my mind is to make the command interactive, so pos-cli env add would prompt user to provide his email and later to provide his staging url..